addons/beehave/debug/tree_node.gd

```diff
@@ -140,8 +138,9 @@ func _check_all_children_on_screen(node: TreeNode) -> void:
 
 
 func _check_for_conflicts(node: TreeNode) -> void:
-	var min_distance = tree_distance + node.item.size.x
+	var min_distance = sibling_distance
```

Comment:
Removed `node.item.size.x` from here and added similar logic to `_get_right_contour` function to account for different sized items